src/calendar/index.tsx

```diff
@@ -91,6 +93,8 @@ class Calendar extends Component<CalendarProps, CalendarState> {
     style: PropTypes.oneOfType([PropTypes.object, PropTypes.array, PropTypes.number]),
     /** Initially visible month. Default = Date() */
     current: PropTypes.any,
+    /** Initially visible month. If changed will initialize the calendar to this value */
+    initialDate: PropTypes.any,
```

Comment:
I think we can kill 2 birds here. 
What if `initialDate` will be a timestamp (number) 
As we discussed before, as part of the big refactor, we'll want to use timestamp for our value source.

Reply:
Do you mean to change 'initialDate' type to a timestamp in preparation for the timestamp support? I will simply convert to XDate at this point...

Reply:
Did we decide if to support timestamp as number or string?

Reply:
Keep the string we have today
and add a ticket to create a util for formatting the value string format